tests/std/tests/P0645R10_text_formatting_custom_formatting/test.cpp

```diff
@@ -58,10 +58,10 @@ struct not_const_formattable_type {
 template <>
 struct std::formatter<basic_custom_formattable_type, char> {
     constexpr basic_format_parse_context<char>::iterator parse(basic_format_parse_context<char>& parse_ctx) {
-        if (parse_ctx.begin() != parse_ctx.end()) {
+        if (parse_ctx.begin() != parse_ctx.end() && *parse_ctx.begin() != '}') {

```

Comment:
Per [[tab:formatter.basic]](https://eel.is/c++draft/tab:formatter.basic), it seems that we should also accept ranges starting with `'}'`.

Reply:
yeah, this is just the case where it's not the last replacement field in the string, so you got `{:.*}`

In fact, I think the '}' case is the _only_ case for us, but it's good to check both anyway.